Orange/widgets/data/owselectcolumns.py

```diff
@@ -171,6 +171,15 @@ class Outputs:
     select_new_features = Setting(True)
     auto_commit = Setting(True)
 
+    UserAdviceMessages = [
+        widget.Message(
+            "<i>Automatically select additional/new features</i> option "
+            "specify whether additional columns in the data that have been on "
+            "the input already before are selected.",
+            "automatically-select-new-features"
+        )
```

Comment:
I read this and couldn't understand. This is really difficult to explain. Perhaps:

```
What happens when the widget receives new data with some additional
columns? If <i>Automatically select additional/new features</i> is on,
they are selected, otherwise not.
```